Packages/MIES/MIES_GuiUtilities.ipf

```diff
@@ -1535,3 +1535,26 @@ Function GetControlCoordinates(win, ctrl, s)
 	s.left   = V_left
 	s.right  = V_left + V_width
 End
+
+/// @brief Get the text (plain or formatted) from the notebook
+Function/S GetNotebookText(win)
+	string win
+
+	ASSERT(WinType(win) == 5, "Passed win is not a notebook")
+
+	Notebook $win getData=1
+
+	return S_Value
+End
```

Comment:
@t-b Do we not use multiple return syntax for functions that only return 1 object?

Reply:
I've opened https://www.wavemetrics.com/forum/general/programming-question-regarding-multiple-return-value-syntax to track some of our earlier questions regarding MR functions.